.github/workflows/tests.yml

```diff
@@ -0,0 +1,79 @@
+name: Tests
```

Comment:
do we have any documentation anywhere on how this file works? and if so, could we add a link to it at the top of this file

Reply:
Added a reference to the actions documentation.